front-end/src/components/search/result/Information.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
```

Comment:
This whole file is more than comments and refactoring; it's entirely new. It looks like work-in-progress, as the server response appears to be mocked...